cli/wallet/wallet.go

```diff
@@ -154,28 +158,31 @@ func NewCommands() []cli.Command {
 				},
 			},
 			{
-				Name:   "create",
-				Usage:  "add an account to the existing wallet",
-				Action: addAccount,
+				Name:      "create",
+				Usage:     "add an account to the existing wallet",
+				UsageText: "neo-go wallet create -w wallet [--wallet-config path]",
+				Action:    addAccount,
 				Flags: []cli.Flag{
 					walletPathFlag,
 					walletConfigFlag,
 				},
 			},
 			{
-				Name:   "dump",
-				Usage:  "check and dump an existing NEO wallet",
-				Action: dumpWallet,
+				Name:      "dump",
+				Usage:     "check and dump an existing NEO wallet",
+				UsageText: "neo-go wallet dump -w wallet [--wallet-config path] [-d]",
+				Action:    dumpWallet,
 				Flags: []cli.Flag{
 					walletPathFlag,
 					walletConfigFlag,
 					decryptFlag,
 				},
 			},
 			{
-				Name:   "dump-keys",
-				Usage:  "dump public keys for account",
-				Action: dumpKeys,
+				Name:      "dump-keys",
+				Usage:     "dump public keys for account",
+				UsageText: "neo-go wallet dump-keys -w wallet [--wallet-config path] [-a address]",
```

Comment:
Replace an old UsageText with `-w wallet [--wallet-config path]` for cases down below? E.g. for ` "export --wallet <path> [--decrypt] [<address>]"`.

Reply:
Done.